code/game/objects/items/storage/backpack.dm

```diff
@@ -606,15 +526,154 @@
+	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
+	new /obj/item/soap/syndie(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/donk
+	name = "\improper Donk Co. duffel bag"
+	desc = "A large duffel bag for holding toys. There is a Donk Co. logo on the back."
+
+/obj/item/storage/backpack/duffelbag/syndie/donk/PopulateContents()
+	new /obj/item/gun/ballistic/automatic/c20r/toy(src)
+	new /obj/item/ammo_box/foambox/riot(src)
+	new /obj/item/ammo_box/foambox/riot(src)
+	new /obj/item/ammo_box/foambox/riot(src)
+	new /obj/item/soap/syndie(src)
+	new /obj/item/grenade/clusterbuster/soap(src)
+	new /obj/item/grenade/chem_grenade/teargas/moustache(src)
+	new /obj/item/disk/nuclear/fake(src)
+	new /obj/item/slimepotion/slime/sentience/nuclear(src)
+	new /obj/item/storage/box/syndie_kit/chameleon/broken(src)
+	new /obj/item/toy/cards/deck/syndicate(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/bioterror
+	name = "bioterrorist duffel bag"
+	desc = "A large duffel bag for holding biohazardous material. Handle with extreme care"
+
+/obj/item/storage/backpack/duffelbag/syndie/bioterror/PopulateContents()
+	new /obj/item/reagent_containers/spray/chemsprayer/bioterror(src)
+	new /obj/item/storage/box/syndie_kit/bioterror(src)
+	new /obj/item/storage/box/syndie_kit/tuberculosisgrenade(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/knight
+	name = "energy knight duffel bag"
+	desc = "A large duffel bag for holding electronic swordsman gear"
+
+/obj/item/storage/backpack/duffelbag/syndie/knight/PopulateContents()
+	new /obj/item/melee/transforming/energy/sword/saber(src)
+	new /obj/item/shield/energy(src)
+	new /obj/item/clothing/shoes/chameleon/noslip(src)
+	new /obj/item/autosurgeon/anti_stun(src)
+	new /obj/item/storage/box/syndie_kit/imp_adrenal(src)
+	new /obj/item/autosurgeon/thermal_eyes(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/cqc
+	name = "cqc duffel bag"
+	desc = "A large duffel bag for holding the secrets of hand to hand combat"
+
+/obj/item/storage/backpack/duffelbag/syndie/cqc/PopulateContents()
+	new /obj/item/book/granter/martial/cqc(src)
+	new /obj/item/clothing/gloves/rapid(src)
+	new /obj/item/soap/syndie(src)
+	new /obj/item/clothing/glasses/phantomthief/syndicate(src)
+	new /obj/item/card/emag(src)
+	new /obj/item/storage/box/syndie_kit/throwing_weapons(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/infiltrator/PopulateContents()
+	new /obj/item/gun/ballistic/automatic/pistol(src)
+	new /obj/item/suppressor(src)
+	new /obj/item/suppressor(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/switchblade(src)
+	new /obj/item/grenade/smokebomb(src)
+	new /obj/item/grenade/smokebomb(src)
+	new /obj/item/card/emag(src)
+	new /obj/item/camera_bug(src)
+	new /obj/item/multitool/ai_detect(src)
+	new /obj/item/implanter/stealth(src)
```

Comment:
Why not give the infiltrator a free chameleon clothes box so they can disguise themselves? its not often bought or valued very high anyways

Reply:
Because nukies shouldn't be getting chameleon kits. It makes infiltration too easy. Literally just waltz into the station looking like a head of staff off the PDA system and bap the captain in the back of the head with CQC. Swap to captain disguise. Done and dusted.